guides/source/active_record_composite_primary_keys.md

```diff
@@ -0,0 +1,303 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
```

Comment:
We seem to be following an existing pattern here but I wonder if Rails should reconsider having such strong message. It's good to encourage reading guides where we think the guides are best presented but also I see no reason to strongly discourage reading them on github if someone finds github more accessible for whatever reason.

Sorry for the off-top!

Reply:
I think it's because otherwise we get PRs to "fix" the guides on github when they are already correct/fine on the guides website.